versioned_docs/version-5.x/basics/upgradeability.md

```diff
@@ -271,6 +271,29 @@ As you can see, delegatee's code looks like a normal ink! Smart Contract with so
 - `addresses` mapping key is identical
 - Constructor does not have any logic, as the code is never instantiated. (It can be, but plays no effect on the execution)
 
+### Delegate dependency locks
+
+The `delegator` contract depends upon the contract code to which it delegates. Since code
```

Comment:
```suggestion
In a delegator contract pattern, one contract delegates calls to another contract. 
Thus it depends upon the contract code to which it delegates. Since code
```